ls1intum/Artemis
# `Bugfix`: Programming exercise grading page shows nan for exercises that give zero automatic points

### Checklist
#### General



#### Client

### Motivation and Context


This PR is related to and closes the issue [6486](https://github.com/ls1intum/Artemis/issues/6486).

### Description

The tops of the left and right columns on the settings page are not aligned in the same position, and there are some unnecessary horizontal lines on the Notifications tab under this page too. This PR aligns them correctly and removes the lines that are not needed.

### Steps for Testing

Prerequisites:
- 1 tutor or Instructor

1. Log in to Artemis
2. Create a programming exercise
3. Open the grading page
4. Mark all weights to 0
5. Confirm that NaN is converted to 0

### Review Progress






#### Code Review
#### Manual Tests


### Screenshots



#### Before
<img width="565" alt="image" src="https://user-images.githubusercontent.com/81355607/235684935-214ac003-e24a-4022-8402-0eaed3b6598f.png">


#### After
<img width="1403" alt="image" src="https://user-images.githubusercontent.com/81355607/235684215-cc64279d-cf76-44c5-984b-7675a3ae6fab.png">
